src/sonic-config-engine/tests/simple-sample-graph.xml

```diff
@@ -20,6 +20,7 @@
         <Multihop>1</Multihop>
         <HoldTime>180</HoldTime>
         <KeepAliveTime>60</KeepAliveTime>
+        <VoQChassisInternal>false</VoQChassisInternal>
```

Comment:
we don't need to set the VoQChassisInternal as false. if VoQChassisInternal  attribute is not present then we assume as False.

Reply:
There are three logical states: 1) VoQChassisInternal is not present, 2) VoQChassisInternal is present and true, 3) VoQChassisInternal is present and false.

While it may not be intended to set it to false, it seemed to me we should test all three cases. However, if that is a problem I guess I can remove it.